index.html

```diff
@@ -395,6 +396,35 @@
       </ul>
     </section>
   </section>
+  <section id="privacy" class="informative">
+    <h2>Privacy and Security</h2>
+    <p>Preload is a declarative fetch primitive that initiates early fetch of
+    resources and separates fetching from resource execution. In effect, it is
+    conceptually similar to initiating a scripted fetch for a resource, but
+    with additional constraints and benefits:</p>
+    <ul>
+      <li>The resource declaration is visible via HTTP headers or HTML markup,
+      which allows the user agent to initiate the fetch earlier, and without
+      blocking on JavaScript execution.</li>
+      <li>The response is not executed and there is no callback to retrieve its
+      payload. The response is retrieved and executed by initiating another
+      fetch (via script, DOM element, etc) that matches the preloaded
+      response.</li>
+      <li>The developer can specify the target context that will consume the
+      response via `as` attribute, which allows the user agent to enforce the
+      relevant CSP policies when initiating the preload fetch. If `as` is
+      omitted, preload defaults to same security and privacy processing as a
+      call to `fetch()` - i.e. subject to `connect-src`.</li>
+    </ul>
+    <p>The site authors are encouraged to take the necessary precautions and
```

Comment:
Should there also be a call to user-agents to respect user's preferences for the context of fetching resources?

That is, for example, if a user has image loading disabled, then so too should a preload relevant to images be ignored.

It may be worth mentioning the passive tracking privacy effect; for example, if a page (such as a search engine results page) indicates preload directives for resources on the subsequent search result pages, then EVEN IF that search result page was served over HTTPS, it may leak context about the nature of the query based on the resources and domains that were preloaded (even if they TOO were delivered over HTTPS). This would not be true in the no-preload case, since the only traffic analysis the attacker could conduct would be in response to actual loads by the user, which would not be anything new.

This is not documenting a *new* attack surface, but merely highlighting for authors and spec reviewers the awareness of the risks of traffic analysis, such that any future solutions that try to mitigate such risks are also aware of the need to find a solution for this spec and not just the 'naive' case.

WDYT?